sim-cli/src/main.rs

```diff
@@ -42,8 +48,11 @@ fn deserialize_f64_greater_than_zero(x: String) -> Result<f64, String> {
 #[derive(Parser)]
 #[command(version, about)]
 struct Cli {
+    /// Path to a directory containing simulation files, and where simulation results will be stored
+    #[clap(long, short, env = "SIM_LN_DATA_DIR", default_value = DEFAULT_DATA_DIR)]
+    data_dir: PathBuf,
     /// Path to the simulation file to be used by the simulator
```

Comment:
Maybe specify that is can either be an absolute path, or relative with respect to `data_dir`?